app/models/registerable_edition.rb

```diff
@@ -14,7 +14,7 @@ def slug
     # to be consistent with Panopticon's slug format.
     panopticon_slug = Whitehall.url_maker.public_document_path(edition).sub(/\A\//, "")
     if edition.deleted?
-      panopticon_slug.sub!(%r{/deleted-([^/]+)$}, '/\1')
+      panopticon_slug.sub!(%r{deleted-([^/]+)$}, '\1')
```

Comment:
That fails the test thusly: 
```
  1) Failure:
RegisterableEditionTest#test_sets_the_correct_slug_for_a_deleted_edition [/Users/boffbowsh/code/whitehall/test/unit/registerable_edition_test.rb:69]:
--- expected
+++ actual
@@ -1 +1 @@
-"government/publications/edition-title"
+"government/publications/deleted-edition-title"
```

which seems to be valid:

```
irb(main):011:0> Artefact.where(slug: %r{^government/publications/deleted-}).count
=> 34
irb(main):012:0> Artefact.where(slug: %r{^government/publications}).count
=> 80354
```

So I think this should be left unanchored.